test/smoke/testApps/TraceLog4j1_2/src/main/resources/ApplicationInsights.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why can't be a common XML used?  This introduces too much redundancy. I was not following every PR's but is it common in the framework to have each test case as an application. If answer is "no" there is no need for separate application. Infact, i would take one step more and reduce the number of test servlets also, club the relevant tests together and reduce the clutter.

Reply:
We do need a separate app because of the log4j library. We want a separate app for each logging framework.

It would be more work that it's worth to try an factor out common XML configs. It's by design to have a separate config for each application. Each application will vary either by config or its compile time dependencies.

Reply:
Makes sense. I get that different logging frameworks will need separate applications as each app can have 1 logging framework. #ByDesign